packages/react/src/ElementFactory/components/ReactElementFactory.tsx

```diff
@@ -30,8 +34,9 @@ export const ReactElementFactory = <
   const rootElement = document.createElement("div");
   const root = createRoot(rootElement);
 
-  // Used when hiding the element. If we were to pass in undefined instead, the element would be immediately cleared, not
-  // leaving time for the fade out animation to complete.
+  // Used when hiding the element. If we were to pass in undefined instead, the
+  // element would be immediately cleared, not leaving time for the fade out
+  // animation to complete.
```

Comment:
do we still need `prevDynamicParams` even if the `getBoundingclientRect` is now always available?

Reply:
Yeah we do, at least for the side menu and slash menus, since they need dynamic params to render as they fade out. Otherwise the side menu throws an error while the drag handle menu is open, and the slash menu renders with no items. Maybe these could be cached like the menu position is, though I feel like that's a bit beyond the scope of this PR.